components/workbenches/workbenches.go

```diff
@@ -115,39 +117,48 @@ func (w *Workbenches) ReconcileComponent(cli client.Client, owner metav1.Object,
 		}
 
 		if platform == deploy.SelfManagedRhods || platform == deploy.ManagedRhods {
-			err := common.CreateNamespace(cli, "rhods-notebooks")
-			if err != nil {
+			if err := cluster.CreateNamespace(cli, "rhods-notebooks"); err != nil {
 				// no need to log error as it was already logged in createOdhNamespace
 				return err
 			}
 		}
-		// Update Default rolebinding
-		err = common.UpdatePodSecurityRolebinding(cli, []string{"notebook-controller-service-account"}, dscispec.ApplicationsNamespace)
-		if err != nil {
+		if err := cluster.UpdatePodSecurityRolebinding(cli, dscispec.ApplicationsNamespace, "notebook-controller-service-account"); err != nil {
 			return err
 		}
 	}
 
-	err = deploy.DeployManifestsFromPath(cli, owner, notebookControllerPath, dscispec.ApplicationsNamespace, ComponentName, enabled)
+	nbPath := notebookControllerPath
```

Comment:
```suggestion
	nbcPath := notebookControllerPath
```
just rename variables to make sense.

Reply:
I will think about something more meaningful, thanks!

Reply:
Thanks, done in 7ed09d5